src/cli/commands/list/tests.rs

```diff
@@ -27,10 +27,6 @@ fn returns_error_in_case_of_failure() {
 	.exec();
 
 	assert!(result.is_err());
```

Comment:
Je te conseille l'excellente lib https://doc.rust-lang.org/std/assert_matches/macro.assert_matches.html
pour assert que des struct matchent bien d'autres. 
Donc la notament tu pourras match le type de ton erreur ainsi que son variant

Reply:
This is a nightly build, wdyt about implementing unstable libs?

Reply:
my bad. https://docs.rs/assert_matches/1.5.0/assert_matches/
This is the crate. It run in stable channel

Reply:
I see no problem using the nightly channel. 
Stable features won't be affected by it. They will stay the same. 
But you will gain access to those nightly features. Those are subject to breaking changes and can even be abandoned. You won't have a guarantee they will ever end up in the stable release. And it's sometimes the only place you can get the feature you need. 
So if you need it, use it. No problem with it.